packages/api/src/tasks/checkAlerts.ts

```diff
@@ -86,7 +85,7 @@ const fireChannelEvent = async ({
   if ((alert.silenced?.until?.getTime() ?? 0) > Date.now()) {
     logger.info({
       message: 'Skipped firing alert due to silence',
-      alertId: alert.id,
+      alertId: alert._id,
```

Comment:
style: `AlertDocument` is preferable here since `id` is a valid virtual field  [docs](https://mongoosejs.com/docs/guide.html#id)